client/src/components/Badge/README.md

```diff
@@ -18,3 +20,4 @@ const props = {
  * `message` (string): The string to display in the badge.
  * `className` (string): Any extra classes to apply for the badge.
  * `inverted` (boolean): If the colours should be inverted.
+ * `stateBadge` (boolean): For use in the context of versioning states (use with inverted).
```

Comment:
It's nice to know in what context this is used, but I would rather know what it actually does.

Reply:
Yeah fair call. I'm not super happy about the way it's ending up either - would love a suggestion!

Basically `inverted` flips the background and text colours, and `stateBadge` or whatever you want to call it will remove the left and right padding from it so it can be used for versioning state badges e.g. "LIVE", "DRAFT" etc.

Reply:
What about `unboxed` or `no-box`?

Reply:
I try to avoid giving things names based on appearance as that might change over time, what happens if we designers add a background colour and padding back as we often want to evolve the design over time.

Reply:
I think we are better off keeping the name "state" as that shouldn't change over time were as its appearance will. I don't think we need to mention "Badge" again in "stateBadge" (maybe its required so it doesn't conflict with another class).

Reply:
`state` is very ambiguous in the context of a react component because you might be talking to the component's internal state or a redux state.

`status` or `isStatusBadge` might be better. Or maybe we create a `<StateBade />` that wrap around the `<Badge/>` component.

Reply:
Perhaps that's the way to go here. `<VersionedStateBadge />` could live in silverstripe/versioned-admin

Reply:
Is versioned-admin a dependency of campaign and CMS? If not you might be introducing some additional coupling.

Reply:
Fair point... which would auto enable history viewer. Maybe I’ll just put it in admin but as a new component